src/main/scala/interplay/Versions.scala

```diff
@@ -2,10 +2,10 @@ package interplay
 
 object ScalaVersions {
   val scala212 = "2.12.15"
-  val scala213 = "2.13.7"
+  val scala213 = "2.13.8"
   val scala3   = "3.1.0"
 }
 
 object SbtVersions {
-  val sbt15 = "1.5.8"
+  val sbt15 = "1.6.1"
```

Comment:
Maybe need to rename the variable to `sbt16`?
```suggestion
  val sbt16 = "1.6.1"
```

Reply:
Yes sure :man_facepalming:, thanks, pushed a commit because there were two places were this needed to be changed.